common/export-common-cartridge-util/src/java/org/sakaiproject/exporter/util/ExportUtil.java

```diff
@@ -0,0 +1,245 @@
+package org.sakaiproject.exporter.util;
+
+import org.apache.commons.lang.StringEscapeUtils;
+import org.sakaiproject.content.api.ContentResource;
+import org.sakaiproject.content.cover.ContentHostingService;
+import org.sakaiproject.util.Validator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ExportUtil {
+	private String siteId;
+	private static Logger log = LoggerFactory.getLogger(CCExport.class);
+	private long nextid = 100000;
+
+	public ExportUtil(String siteId) {
+		this.siteId = siteId;
+	}
+
+	public Resource getResource(String sakaiId, String location) {
+		return getResource(sakaiId, location, null);
+	}
+
+	public Resource getResource(String sakaiId, String location, String use) {
+		Resource res = new Resource();
+		res.setSakaiId(sakaiId);
+		res.setResourceId(getResourceId());
+		res.setLocation(location);
+		res.setDependencies(new HashSet<>());
+		res.setUse(use);
+		res.setIslink(false);
+		res.setIsbank(false);
+		return res;
+	}
+
+	String getFilePath(ContentResource contentResource, boolean encode) {
```

Comment:
I'd normally pass in the actual ID as then it's easier to test and you're not binding against the ContentResource object.